mslib/utils/_tests/tests.py

```diff
@@ -64,6 +65,7 @@ def test_load_settings(self):
         assert settings["foo"] == "bar"
 
 
+@pytest.mark.skip("To be done for new refactor")
 class TestConfigLoader(object):
     """
```

Comment:
tests needs also be splitted in multiple modules
preferable we should have "module"  and "test_module"
the current tests.py is a placeholder from the long file.

Reply:
I was planning to do that once we decided if the refactor was okay.